spec/models/event_spec.rb

```diff
@@ -20,8 +20,8 @@
     context "creating new event without remote_id" do
       it "sends call to update NationBuilder" do
         expect(Integration::NationBuilder).to receive(:create_event)
-          .with({ attributes: event_attributes })
-        FactoryGirl.create(:event, starts_at: start, ends_at: start + 1.hour)
+          .with({ attributes: event_attributes }) { 1 }
```

Comment:
Redundant curly braces around a hash parameter.<br>Place the . on the previous line, together with the method call receiver.